nva-datamodel-java/src/test/java/no/unit/nva/OntologyTest.java

```diff
@@ -64,18 +66,27 @@ void shouldContainDistinctDescriptions() {
         assertThat(duplicatesMessage, distinctValues, is(equalTo(ontologyValues)));
     }
 
-    @Test
-    void shouldContainEveryVisibleClassOfModel() {
+    @ParameterizedTest
+    @MethodSource("provideModelClassesArguments")
+    void shouldContainEveryVisibleClassOfModel(String modelClass) {
         var ontologyClasses = extractClassesFromOntology();
         var modelClasses = new ArrayList<>(getModelClasses()).toArray(String[]::new);
         assertThat(ontologyClasses, hasItems(modelClasses));
     }
 
-    @Test
-    void shouldContainEveryVisiblePropertyOfModel() {
+    @ParameterizedTest
+    @MethodSource("provideModelPropertiesArguments")
+    void shouldContainEveryVisiblePropertyOfModel(String modelProperty) {
         var ontologyProperties = extractPropertiesFromOntology();
-        var modelProperties = new ArrayList<>(getModelProperties()).toArray(String[]::new);
-        assertThat(ontologyProperties, hasItems(modelProperties));
+        assertThat(ontologyProperties, hasItem(modelProperty));
+    }
+
+    public static Stream<Arguments> provideModelPropertiesArguments() {
```

Comment:
Call this modelPropertyProvider and keep it at the top of the class — this convention should probably be enforced by some rule, but isn't, alas.

Reply:
fixed